english/ghostman/ghost.md

```diff
@@ -0,0 +1,3 @@
+Welcome dear fellas. You have entered the ghostland. You no longer exist. And here comes Shrek from Neverland. For now enjoy the trailer from his movie [Shrek] (https://www.youtube.com/watch?v=W37DlG1i61s).
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.